brig/cmd/brig/commands/build_logs.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Azure/brigade/pkg/brigade"
+	"github.com/Azure/brigade/pkg/storage"
+	"github.com/Azure/brigade/pkg/storage/kube"
+)
+
+const (
+	logHeader      = "\n==========[  %s  ]==========\n"
+	buildLogsUsage = `Show log(s) for a build
+
+Print the logs for a build, and (optionally) the jobs executed as part of the
+build.
+`
+)
+
+var logsAll bool
+
+func init() {
+	buildLogs.Flags().BoolVarP(&logsAll, "all", "a", false, "Show job logs as well as the worker log")
```

Comment:
This flag name should probably be `--jobs`.

`--all` seems like I should get logs for all workers `brig build logs --all`

Reply:
Fixed